### ISSUE TITLE
[Documentation] Date picker documentation examples teach buggy way to get ISO date

### ISSUE THREAD

**Issue by hasiili** [vuetifyjs/vuetify#13815]
### Problem to solve
Date picker documentation examples use this code to get date in ISO format
new Date().toISOString().substr(0, 10)

Depending on user's time zone and time of day, this is sometimes correct and sometimes off by a day. I would bet that the developers intention is mosly to get the current date on their time zone, not date on UTC.

### Proposed solution
Documentation should either use hard coded date like '2021-06-18' or some way to get local date

**Comment by KaelWD** [comment]
Goddamn the Date API sucks:
```js
(new Date(Date.now() - tzoffset)).toISOString().substr(0, 10)
```

With Temporal instead:
```js
Temporal.now.plainDateISO().toString()
```

**Comment by MajesticPotatoe** [comment]
resolved in ca92bb5
